### DISCUSSION TITLE
Sourdough Compatibility

### DISCUSSION THREAD

**Discussion by deepslater** [Lgmrszd/CompressedCreativity#43] [Ideas]
As it stands, PCR's wheat flour can't be used to craft anything except Sourdough - and Create's wheat flour can't be made into Sourdough either.

My idea is as follows:
1) Add a mixer recipe for Sourdough from 1 #forge:flour and 125mb Yeast Culture, consistent with the current PCR recipe.
2) Optionally, either remove PCR's wheat flour and replace it in PCR recipes with Create wheat flour, or make PCR wheat flour usable in other flour recipes.

This is something I've done using KubeJS but that's obviously not something I can commit in a pull request, and I'm pretty inexperienced.

**Comment by Lgmrszd** [comment]
Thanks for the idea! It makes sense, I'll look into that. Possibly the items are just missing tags
